src/components/computed-properties.vue

```diff
@@ -195,7 +198,28 @@ export default {
     },
     'add.property': {
       handler() {
-        this.validateDuplicate();
+        if (this.$refs.propName && this.$refs.propName.validator) {
+          this.$refs.propName.validator.messages.messages.not_in = this.$t('Property already exists.');
+        }
+        let data = '';
+        this.current.forEach(item => {
+          if (item.property === this.add.property && item.id !== this.add.id) {
+            data += data ? ',' + item.property : 'not_in:' + item.property;
+          }
+        });
+        this.rulePropName = 'required' + (data ? '|' + data : '');
+      },
+      deep: true,
```

Comment:
I don't think you need to specify `deep: true` for any of the `add` property watchers; all of these properties are primitive values (not objects), so there's no actual _deep_ value to watch.

Reply:
fixed